src/components/permissions/routes.ts

```diff
@@ -20,12 +27,31 @@ const routeLogin = async (request: any, reply: any) => {
   if (!(await bcrypt.compare(password, passwordHash))) return reply.send({ success: false })
 
   // Login successful
+  const userInfo = await getUserInfo(username)
+  const orgList = await databaseConnect.getUserOrgs(userInfo.user.userId)
   reply.send({
     success: true,
-    ...(await getUserInfo(username)),
+    ...userInfo,
+    orgList,
   })
 }
 
+// Authenticates user and checks they belong to requested org (id). Returns:
+// - userInfo (including orgId and orgName)
+// - template permissions
+// - JWT (with orgId included)
+const routeLoginOrg = async (request: any, reply: any) => {
+  const { userId, orgId } = request.body || { userId: '', orgId: '' }
```

Comment:
do we need userId in the body ?

Reply:
You mean should it just come from the JWT?

Yeah I guess that would be fine. OrgId too then, I guess. So the body can be empty?

Reply:
Ah, not the orgId, it's not in JWT yet. Got it.